sdk/appcenter/src/test/java/com/microsoft/appcenter/channel/DefaultChannelTest.java

```diff
@@ -67,6 +66,8 @@
 @PrepareForTest(System.class)
 public class DefaultChannelTest extends AbstractDefaultChannelTest {
 
+    private final String START_TIMER_PREFIX = "START_TIMER_PREFIX_";
```

Comment:
It was fine to use that one. The warning was on `MINIMUM_TRANSMISSION_INTERVAL` that isnt used in tests.
START_TIMER_PREFIX could be VisibleForTest here.